index.js

```diff
@@ -441,6 +441,20 @@
   //. ```
   Maybe.prototype.of = def('Maybe#of', [a], Maybe.of);
 
+  //# Maybe#reduce :: Maybe a ~> (b -> a -> b) -> b -> b
+  //.
+  //. Takes a function and an initial value, returning the result of applying
+  //. the function to the initial value and the value of the Just. If `this`
+  //. is a Nothing, then the initial value is simply returned instead.
```

Comment:
Perhaps it would be clearer to make each case a list item:

> Takes a function and an initial value of any type, and returns:
>
>   - the initial value if `this` is a Nothing; otherwise
>   - the result of applying the function to the initial value and this Just's value.

Do you think it's accurate to describe the value as an *initial* value?